composer.json

```diff
@@ -16,17 +16,24 @@
     }
   ],
   "require": {
-    "php": ">=7.0",
-    "symfony/filesystem": "~3.0|~4.0",
-    "symfony/dependency-injection": "~3.0|~4.0",
-    "symfony/security": "~3.0|~4.0",
-    "symfony/config": "~3.0|~4.0",
-    "symfony/event-dispatcher": "~3.0|~4.0",
-    "symfony/http-foundation": "~3.0|~4.0",
-    "symfony/http-kernel": "~3.0|~4.0",
-    "symfony/twig-bundle": "~3.0|~4.0",
+    "php": ">=7.3",
+    "symfony/filesystem": "~4.4|~5.0",
```

Comment:
What makes you remove Symfony 3 support?

Reply:
Multiple reasons for no longer supported Sf3:
- The Mailer is not supported
- The new events are not available

It eases the maintenance.